index.html

```diff
@@ -84,7 +85,13 @@ <h3>About Protectors of the Salish Sea</h3>
             </p>
           </div>
         </div>
+      	<div id="contact-form" class="section">
+          <div id='can-form-area-climate-emergency-2020-contact'>
```

Comment:
Probably best to keep `'` or `"` consistent throughout.

Reply:
We'll also have to add some styling on this so that the join button matches the accent color on the rest of the page.